deep-learning-tools/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+type: Opaque
+data:
+  notebook_config.py: {{ include "notebook-config" . }}
```

Comment:
Nice work! It otherwise looks (and works) nicely :)

You can bump the version in the Chart.yaml file as well, if you want